resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/adaptive/LimitAdapter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ *  Copyright 2019: Mahmoud Romeh
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package io.github.resilience4j.bulkhead.adaptive;
+
+import java.time.Duration;
+import java.util.function.Consumer;
+
+import io.github.resilience4j.bulkhead.Bulkhead;
+import io.github.resilience4j.bulkhead.BulkheadConfig;
+import io.github.resilience4j.bulkhead.event.BulkheadLimit;
+import io.github.resilience4j.core.lang.NonNull;
+
+
+/**
+ * Limit adopter interface for adoptive bullhead {@link AdaptiveBulkhead}
```

Comment:
spelling - `Limit adapter interface for adaptive bulkhead {@link AdaptiveBulkhead}`